Bottom: f068990f0331b6a4c4fa7eadc64d3d51d3087be2
Top:    8b3ca1fa6e1d84d2419b4bb7781006a4b01ae60c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-01-03 22:56:53 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
